handbook/engineering/developer-insights/extensibility/index.md

```diff
@@ -129,6 +136,10 @@ The team follow's the [default code review guidelines](https://docs.sourcegraph.
 1. If the author would like any of the requested reviewers to merge the PR after approval they add the label `merge-on-any-approve`
 1. If the author would like their PR to be merged once all of the requested reviewers have approved it they add the label `merge-on-all-approve`
 
+#### Cross-functional reviews
+
+Engineers will often need reviews from teammates from other functions. For example, a pull request can benefit from visual review from a designer. In such cases, the author should be sure to request the PR review as well as tag the requested cross-functional reviewer(s) in their PR description, along with what exactly they need reviewed and when they need it to be reviewed by (e.g. "@Joelkw can you review the banner copy by the end of the week?").
```

Comment:
Might be worth linking to https://about.sourcegraph.com/handbook/communication/seeking-and-giving-feedback#seeking-feedback, which applies here

Reply:
Thanks!